include/color_spinor_field.h

```diff
@@ -736,23 +501,8 @@ namespace quda {
                    MemoryLocation location[2 * QUDA_MAX_DIM], MemoryLocation location_label, bool spin_project,
                    double a = 0, double b = 0, double c = 0, int shmem = 0);
 
-    /**
-      Initiate the gpu to cpu send of the ghost zone (halo)
-      @param ghost_spinor Where to send the ghost zone
-      @param dim The lattice dimension we are sending
-      @param dir The direction (QUDA_BACKWARDS or QUDA_FORWARDS)
-      @param stream The array of streams to use
-      */
-    void sendGhost(void *ghost_spinor, const int dim, const QudaDirection dir, const qudaStream_t &stream);
-
-    /**
-      Initiate the cpu to gpu send of the ghost zone (halo)
-      @param ghost_spinor Source of the ghost zone
-      @param dim The lattice dimension we are sending
-      @param dir The direction (QUDA_BACKWARDS or QUDA_FORWARDS)
-      @param stream The array of streams to use
-      */
-    void unpackGhost(const void *ghost_spinor, const int dim, const QudaDirection dir, const qudaStream_t &stream);
+    // fuse with above
```

Comment:
should this be taken care of in this PR?

Reply:
I consider the packing aspects as part of the comms cleanup, which as we've discussed, is deferred for a follow up PR